### PR TITLE
feat(telephony.alias): handle ddi numbers

### PR DESCRIPTION
**Pull request by JeremyDec**
MANAGER-2043

### CODE REVIEW

File: client/app/telecom/telephony/alias/configuration/feature/redirect/telecom-telephony-alias-configuration-redirect.controller.js [hunk 1]
```diff
@@ -55,15 +57,27 @@ angular.module('managerApp').controller('TelecomTelephonyAliasConfigurationRedir
...
         this.destinationUsedAsPresentation = false;
       }
+
+      return this.tucVoipServiceLine.getPhone({
+        billingAccount: service.billingAccount,
+        serviceName: service.serviceName,
+      }).then(() => {
+        this.featureTypeToUse = 'redirect';
+      }).catch(() => {
+        this.featureTypeToUse = 'ddi';
```
Comment (marie-j) [inline client/app/telecom/telephony/alias/configuration/feature/redirect/telecom-telephony-alias-configuration-redirect.controller.js]: I suppose this works if it's a `404` but what if it's a `504` for instance ?

File: client/app/telecom/telephony/alias/configuration/telecom-telephony-alias-configuration.js [hunk 2]
```diff
@@ -8,10 +8,14 @@ angular.module('managerApp').config(($stateProvider) => {
         controllerAs: '$ctrl',
       },
       'featureView@telecom.telephony.alias.configuration': {
-        templateProvider: alias => `<div data-ng-include="::'app/telecom/telephony/alias/configuration/feature/${alias.featureType}/telecom-telephony-alias-configuration-${alias.featureType}.html'"></div>`,
+        templateProvider: (alias) => {
+          const featureType = alias.featureType === 'ddi' ? 'redirect' : alias.featureType;
+          return `<div data-ng-include="::'app/telecom/telephony/alias/configuration/feature/${featureType}/telecom-telephony-alias-configuration-${featureType}.html'"></div>`;
+        },
```
Comment (marie-j) [inline client/app/telecom/telephony/alias/configuration/telecom-telephony-alias-configuration.js]: Could this be replaced by a nested view instead ?

File: client/app/telecom/telephony/alias/configuration/feature/redirect/telecom-telephony-alias-configuration-redirect.controller.js (hunk 1 above)
Comment (JeremyDec) [reply client/app/telecom/telephony/alias/configuration/feature/redirect/telecom-telephony-alias-configuration-redirect.controller.js]: Good point :)

**Comment by jleveugle** [comment]
Hello @JeremyDec,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Pull request closed without merging**